### PR TITLE
Update Disclosure Extractor

### PR DESCRIPTION
**Pull request by flooie**
Add support for vector based PDFs.

### CODE REVIEW

File: cl/disclosures/tasks.py [hunk 1]
```diff
@@ -372,6 +387,8 @@ def generate_or_download_disclosure_as_pdf(
     :param pdf_url: The URL of PDF in S3
     :return: Response containing PDF
     """
+    logger.info(pdf_url)
+    logger.info(data)
```
Comment (mlissner) [inline cl/disclosures/tasks.py:391]: Are these debug lines, or did you intend them here? If the latter, we should provide more context than just the variable values, and probably put them on one line.

**Review by mlissner — COMMENTED** [review]
Looks fine. One small q.

File: cl/disclosures/tasks.py (hunk 1 above)
Comment (flooie) [reply cl/disclosures/tasks.py:391]: these should've been removed.

**Comment by mlissner** [comment]
Than kyou.

**Pull request merged**